app/views/gobierto_data/layouts/_navigation.main.html.erb

```diff
@@ -1,3 +1,4 @@
-<div class="main-nav-item<%= class_if(" active", current_module == "gobierto_data") %>">
+# REVIEW: get an element ID to inject vue-router behavior
```

Comment:
This is not a valid comment, and it's visible in the HTML

![Screenshot 2020-05-15 at 20 03 42](https://user-images.githubusercontent.com/17616/82081766-30e2c080-96e7-11ea-8b4d-ec551977cbba.png)